testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_logging.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from CBLClient.FileLogging import FileLogging
+from keywords.utils import log_info
+
+log_level_dict = {
+    "debug": 0,
+    "verbose": 1,
+    "info": 2,
+    "warning": 3,
+    "error": 4,
+}
+
+
+@pytest.mark.sanity
+@pytest.mark.listener
+@pytest.mark.database
+@pytest.mark.parametrize("log_level, plain_text, max_size, max_rotate_count", [
+    ("debug", False, 0, 0),
+    ("verbose", False, 0, 0),
+    ("info", False, 0, 0),
+    ("warning", False, 0, 0),
+    ("error", False, 0, 0),
+    ("debug", True, 3 * 512 * 1000, 3),
+    ("verbose", True, 4 * 512 * 1000, 4),
+    ("info", True, 5 * 512 * 1000, 5),
+    ("warning", True, 6 * 512 * 1000, 6),
+    ("error", True, 7 * 512 * 1000, 7),
```

Comment:
just wondering, do you know what it the max rotate count? Is it limited or it can be any number . If there is no limitation, can u check with 1000 count too

Reply:
Sure, I'll check and configure something for that.

Reply:
added